db/schema.rb

```diff
@@ -41,6 +41,20 @@
     t.index ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true
   end
 
+  create_table "book_suggestions", force: :cascade do |t|
+    t.string "synopsis"
+    t.float "price"
+    t.string "author", null: false
+    t.string "title", null: false
+    t.string "link", null: false
+    t.string "editor", null: false
+    t.string "year", null: false
+    t.bigint "user_id"
+    t.datetime "created_at"
```

Comment:
Usually time stamp are like this
```suggestion
     t.datetime "created_at", null: false
```

Reply:
Yes, but I have previously pushed the table creation without timestamps, so if I add the not null constraint without a default value, that migration will fail if there are book suggestions in the database. In this case that would not cause any issue since i'm the only one coding in this repo, but in a real project that could cause a problem. How do you suggest to face this issue ?